src/components/frame/Extensions/pages/LazyConsensusPage/LazyConsensusPage.tsx

```diff
@@ -48,72 +49,56 @@ const LazyConsensusPage: FC = () => {
     !extensionData.isDeprecated &&
     extensionData?.extensionId === Extension.VotingReputation;
 
+  const activeInstalls = Number(
```

Comment:
is repeated twice, please move it to <Action Buttons /> component, to avoid conflicts please not change the name of component

Reply:
You can pass extensionId to hook as param and just return installs for this extension

Reply:
Done